### ISSUE TITLE
Podcast migration from soundcloud

### ISSUE THREAD

**Issue by KevRM** [CitizensBudgetCommission/cbcny-d8#197]

**Comment by peterserfozo** [comment]
As I see, there are deprecated migrations and migrations groups in the current config, these should be removed. After that, the podcast migration should be reset and restarted.
So the proposed workflow is:
1. Remove deprecated migration configuration & objects
2. Deploy the new config
3. Reset & restart the podcast migration by hand
4. Further investigation if not resolved

**Comment by peterserfozo** [comment]
Fixed the podcast migration config.

You can start a new migration here: https://cbcny.org/admin/structure/migrate/manage/cbc_podcast/migrations
When both migrations are in "Idle" status you can start a new migration. See the attached screenshot:
![screencapture-cbcny-org-admin-structure-migrate-manage-cbc-podcast-migrations-2020-07-21-12_04_08](https://user-images.githubusercontent.com/22172388/88041366-8fdf0e80-cb4a-11ea-9c38-1559b76be7fd.png)

Sometimes a migration process may fail to stop cleanly, and be left stuck in an Importing status. In this case, you should reset the migration. To do this, visit the problematic migration and select the option "Reset" and click "Execute". It will reset the stuck migration. Then you can select the "Import" option and execute the migration and it will work properly.

tl;dr
If one of the migration is left stuck in an "Importing" status then execute a "Reset" operation first then an "Import" operation.
